lib/runtime/entry.js

```diff
@@ -115,9 +115,9 @@ function addNamespace(ns) {
 
   if (this.loaded) {
     Object.seal(ns);
-  } else {
-    this.nsObjects.push(ns);
   }
+
+  this.namespaces.push(ns);
```

Comment:
This also tracks the ns objects even after load in case we want to query them for some reason.